pytest_rpc/__init__.py

```diff
@@ -29,6 +29,22 @@
             'JOB_NAME',
             'MOLECULE_TEST_REPO',
             'MOLECULE_SCENARIO_NAME']
+PYTEST_ENV_VARS = ['BUILD_URL',
```

Comment:
I think this issue may be pointing to a flaw in the design of the ENV_VAR mapping. The difference between these sets of variables has less to do with the choice of tools and more to do with the test environments they are collecting data on. The `ENV_VARS` specifically map to rpc_gating centric env_vars. The `PYTEST_ENV_VARS` appear to be more generally structured around a mono-repo build/test environment.

In the immediate term, the names of these variables should be more descriptive to their environment.

In the long term, we may want to take another look at how these collected. Perhaps it should be the test suites responsibility to define variables associated with its run.

Reply:
I agree with your assertion that this is about
>the test environments they are collecting data on
What we care about is the intersection of `project`, `testing tools`, and `testing environment`.  We are definitely conflating things at the moment, but in the interest of not doing everything at once it seems like this is the way to slice off the smallest change.  The other side of this is that ZigZag cares even more about these things since it has control over how the data will be displayed to the end user.